apps/meteor/ee/app/livechat-enterprise/server/methods/getUnitsFromUserRoles.ts

```diff
@@ -1,15 +1,22 @@
 import { Meteor } from 'meteor/meteor';
+import mem from 'mem';
 
 import { hasAnyRoleAsync } from '../../../../../app/authorization/server/functions/hasRole';
 import LivechatUnit from '../../../models/server/models/LivechatUnit';
 
+export async function getUnitsFromUserRoles(user: Meteor.User | null): Promise<{ [k: string]: any }[] | undefined> {
```

Comment:
can we change this function to receive only `userId`? it seems it uses only the `_id`..

the reason I'm asking is that since `Meteor.User` is a complex object, in order to memoize the result correctly, the input should be stringified first.